File: Insurance-2306226864-fe/src/enums/claimStatus.enum.ts
export enum ClaimStatusEnum {
  WAITING_FOR_REVIEW = 'WAITING_FOR_REVIEW',
  ACCEPTED = 'ACCEPTED',
  REJECTED = 'REJECTED'
}

export const ClaimStatusLabels: Record<ClaimStatusEnum, string> = {
  [ClaimStatusEnum.WAITING_FOR_REVIEW]: 'Waiting for Review',
  [ClaimStatusEnum.ACCEPTED]: 'Accepted',
  [ClaimStatusEnum.REJECTED]: 'Rejected'
}

export const ClaimStatusColors: Record<ClaimStatusEnum, string> = {
  [ClaimStatusEnum.WAITING_FOR_REVIEW]: 'bg-yellow-100 text-yellow-800',
  [ClaimStatusEnum.ACCEPTED]: 'bg-green-100 text-green-800',
  [ClaimStatusEnum.REJECTED]: 'bg-red-100 text-red-800'
}

File: Insurance-2306226864-be/src/main/java/apap/ti/_5/Insurance_2306226864_be/enums/ClaimStatusEnum.java
package apap.ti._5.Insurance_2306226864_be.enums;

public enum ClaimStatusEnum {
    WAITING_FOR_REVIEW("Waiting for Review"),
    ACCEPTED("Accepted"),
    REJECTED("Rejected");

    private final String displayName;

    ClaimStatusEnum(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }

    @Override
    public String toString() {
        return displayName;
    }
}